symfony/client/src/orangehrmLeavePlugin/pages/LeaveAssign.vue

```diff
@@ -290,5 +372,29 @@ export default {
       return id && (id === 3 || id === 4);
     },
   },
+
+  watch: {
+    'leave.employee': function(employee) {
+      if (!employee) {
+        return;
+      } else {
+        this.http.request({
+             method: 'GET',
+             url: 'api/v2/pim/employees/' + this.leave.employee?.id + '/work-shift',
+          })
+          .then(response => {
+            const {data} = response.data;
+            if (data) {
+              this.workShift = data;
+            } else {
+              this.$toast.error({
```

Comment:
Please get confirmation from this as the intended behaviour from business team and please check the toast message's message and type.

Reply:
done